deploy/crds/openebs_v1alpha1_openebsinstalltemplate_cr.yaml

```diff
@@ -159,16 +207,16 @@ spec:
   cstor:
     pool:
       image: quay.io/openebs/cstor-pool
-      imageTag: 2.11.0
+      imageTag: 2.12.2
     poolMgmt:
       image: quay.io/openebs/cstor-pool-mgmt
-      imageTag: 2.11.0
+      imageTag: 2.12.2
     target:
       image: quay.io/openebs/cstor-istgt
-      imageTag: 2.11.0
+      imageTag: 2.12.2
     volumeMgmt:
       image: quay.io/openebs/cstor-volume-mgmt
```

Comment:
could you change all the `quay.io` images to docker images please. It should be `openebs/cstor-volume-mgmt` instead of `quay.io/openebs/cstor-volume-mgmt`